definitions/ext-argocd/definition.yml

```diff
@@ -0,0 +1,65 @@
+domain: EXT
+type: ARGOCD
+synthesis:
+  rules:
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_app_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_cluster_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_git_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_kubectl_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_redis_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: grpc_server_
+
+  tags:
+    label.app.kubernetes.io/name:
```

Comment:
This will be the name of the entity so is a field we can already filter by. What is the intent on having it also as a tag?

Reply:
Good call out.  I removed it as it wasn't necessary for this.